components/pedigree/resources/src/main/resources/pedigree/view/workspace.js

```diff
@@ -139,17 +140,17 @@ define([
             // set display:block
             svgText = svgText.replace(/(<svg[^<>]+style=")/g, "$1display:block; ");
```

Comment:
It is definitely inefficient. But it is done once on save, and once on print, so maintainability/simplicity/speed of development is more important than the speed of regexp evaluation which is 0 for all practical purposes here.